doc/mgr/dashboard.rst

```diff
@@ -287,7 +287,11 @@ More details can be found on the `documentation <http://docs.ceph.com/docs/maste
         grafana-cli plugins install vonage-status-panel
         grafana-cli plugins install grafana-piechart-panel
 
-#. Add the Dashboards to Grafana by importing them
+#. Add the Dashboards to Grafana by importing them::
+	
+	wget https://raw.githubusercontent.com/ceph/ceph/master/monitoring/grafana/dashboards/<Dashboard-name>.json
```

Comment:
Just wondering if we should add a wget command (as example) that directly includes the ceph-cluster.json (because there is currently only this Grafana dashboard anyway and people can just copy-paste it from our documentation instead of visiting the site first).

Reply:
Ack, will add an example for currently added json.